src/Sylius/Bundle/UiBundle/Resources/views/Form/theme.html.twig

```diff
@@ -40,7 +40,7 @@
 
 {% block money_widget -%}
     <div class="ui labeled input">
-        <div class="ui label">{{ sylius_currency_symbol() }}</div>
+        <div class="ui label">{{ shopperContext.currencyCode|sylius_currency_symbol }}</div>
```

Comment:
Shouldn't we use default currency here? Isn't this widget used in admin where we don't have shopper context?